amivapi/settings.py

```diff
@@ -116,3 +116,37 @@
     'mtec',
     'gess'
 ]
+
+# Config for swagger (/docs)
+# TODO: should be linked to the setup.py information in the package
+SWAGGER_INFO = {
+    'title': 'AMIVAPI',
+    'version': '0.1-dev',
```

Comment:
The way many popular packages do it is to include a `__version__ = '0.1-dev'` in `amivapi/__init__.py` and read that from `setup.py` via a regular expression (like `r'__version__\s+=\s+(.*)'`).

Examples:
* [flask](https://github.com/pallets/flask/blob/master/setup.py#L52-L56)
* [requests](https://github.com/kennethreitz/requests/blob/master/setup.py#L52-L54)
* [SQLAlchemy](https://github.com/zzzeek/sqlalchemy/blob/master/setup.py#L112-L118)

There are also [other possibilities](https://packaging.python.org/single_source_version/), but IMHO the above approach works best. (See the bullet points in [this StackOverflow answer](https://stackoverflow.com/a/7071358) for some reasons why to use regular expressions for this task.)